build.sbt

```diff
@@ -7,82 +7,83 @@ import play.sbt.PlayImport.PlayKeys.*
 import sbtversionpolicy.withsbtrelease.ReleaseVersion
 
 ThisBuild / scalaVersion := "3.3.5"
-ThisBuild / crossScalaVersions := Seq(
-  scalaVersion.value,
-  "2.13.16"
-)
```

Comment:
This setting comes out, because we no longer want _all_ projects to be cross-compiled with both Scala 2 & Scala 3 - the key rotation script is just for Scala 3.